src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+
+import classNames from 'classnames';
+
+type Props = {
+  errorMessage: string
+};
+
+export const Notification: React.FC<Props> = ({ errorMessage }) => {
+  const [isHidden, setIsHidden] = useState(false);
+
+  const autoHide = () => {
+    setTimeout(() => {
+      setIsHidden(true);
+    }, 3000);
+  };
+
+  autoHide();
+
+  return (
+    <div
+      className={classNames(
+        'notification',
+        'is-danger',
+        'is-light',
+        'has-text-weight-normal',
+        { hidden: isHidden },
+      )}
+    >
+      <button // eslint-disable-line jsx-a11y/control-has-associated-label
```

Comment:
Same us previous comment,

Reply:
https://github.com/mate-academy/react_todo-app-add-and-delete/pull/327/files#r1122891646